src/hub.js

```diff
@@ -278,6 +278,7 @@ async function updateEnvironmentForHub(hub) {
   let isLegacyBundle; // Deprecated
 
   const environmentScene = document.querySelector("#environment-scene");
+  const scene = document.querySelector("a-scene");
```

Comment:
I might change this to `sceneEl` since `scene` is pretty overloaded in this function.